### ISSUE TITLE
A file imported by reference and then normally 

### ISSUE THREAD

**Issue by SomMeri** [less/less.js#2716]
Main test.less:
````less
div-direct {
  @import (reference) "file.less";
}

show-all-content {
  @import "file.less";
}
````

file.less:
````less
/*
  tralala
*/
.fix {
  fix: fix;
}
.something {
  @import (reference) "file-2.less";
}
````

file-2.less:
````less
/* tralala nested file */
should {
  be: invisible;
}
````

compiles into:
````less
div-direct .fix {
  fix: fix;
}
div-direct .something should {
  be: invisible;
}
````

It should be (plus or minus comments):
````less
show-all-content .fix {
  fix: fix;
}
````

**Comment by seven-phases-max** [comment]
Note the second "file.less" import has no effect until you add `multiple` there. So I suppose the resulting CSS is actually an example of another issue, #1896 probably?

**Comment by SomMeri** [comment]
@seven-phases-max You are right, maybe output should be empty. (Should imports with and without reference be seen as copies for multiple purpose?).

I am trying to fully fix import by reference now, so that it works correctly in the end. If I succeed I hope to close all (or most) reference related bugs.

**Comment by SomMeri** [comment]
This should be fixed by #2729 .
